cozy/ui/widgets/book_card.py

```diff
@@ -118,6 +118,10 @@ def set_playing(self, is_playing):
     def update_progress(self):
         self.play_button.progress = self.book.progress / self.book.duration
 
+    def reset(self) -> None:
+        # self.book.position = 0
```

Comment:
```suggestion
```
Since we don't want to reset the playback position, this commented code will not be necssary.